### PR TITLE
TG-1890:@youwol/platform ->upgrade to rxjs#7 + rx-vdom

### PR DESCRIPTION
**Pull request by jehadmelad**
TG-1890

### CODE REVIEW

File: src/app/modals/launchpad/actions.view.ts [hunk 1]
```diff
@@ -33,19 +38,23 @@ export class SideAppActionsView implements VirtualDOM {
                 modalState: params.modalState,
                 app: params.app,
             }),
-            child$(
-                OsCore.RequestsExecutor.getAsset(
+            // {
```
Comment (youwol-guillaume) [inline src/app/modals/launchpad/actions.view.ts:41]: Remove commented code

File: src/app/modals/launchpad/actions.view.ts [hunk 2]
```diff
@@ -165,32 +189,45 @@ class SideAppFavoriteAction implements VirtualDOM {
...
 function installMarked$() {
     return from(
-        cdnClient.install({
+        webpmClient.install({
             modules: [`marked#${setup.runTimeDependencies.externals.marked}`],
         }) as unknown as Promise<{ marked: typeof Marked }>,
-    ).pipe(map(({ marked }) => marked))
+    ).pipe(
+        map(({ marked }) => marked),
+        tap((v) => console.log('type Of !! ', typeof v)),
```
[the remark this answers is not in the document]
Comment (jehadmelad) [reply src/app/modals/launchpad/actions.view.ts:197]: noted.

File: src/app/modals/launchpad/actions.view.ts (hunk 1 above)
Comment (jehadmelad) [reply src/app/modals/launchpad/actions.view.ts:41]: noted

File: src/app/modals/profiles/profiles.state.ts [hunk 3]
```diff
@@ -18,15 +18,14 @@ import { CdnSessionsStorage } from '@youwol/http-clients'
...
+import { TsCodeEditorModule } from '@youwol/rx-code-mirror-editors'
+
+import * as webpmClient from '@youwol/webpm-client'
 import * as httpClients from '@youwol/http-clients'
-import * as fluxView from '@youwol/flux-view'
 
 const cmInstall = {
     modules: [
-        `@youwol/fv-code-mirror-editors#${setup.runTimeDependencies.externals['@youwol/fv-code-mirror-editors']}`,
+        `@youwol/rx-code-mirror-editors#${setup.runTimeDependencies.externals['@youwol/rx-code-mirror-editors']}`,
```
Comment (youwol-guillaume) [inline src/app/modals/profiles/profiles.state.ts:28]: You can simplify by using 'as codeMirrorEditors' & remove aliases

File: src/app/running-apps/running-app.view.ts [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
     /**
      * @group Immutable DOM Constants
      */
-    public readonly class: Stream$<OsCore.RunningApp, string>
+    public readonly tag = 'div'
+    /**
+     * @group Immutable DOM Constants
+     */
+    public readonly class: AttributeLike<string>
+    // Stream$<OsCore.RunningApp, string>
```
Comment (youwol-guillaume) [inline src/app/running-apps/running-app.view.ts:17]: remove commented code

File: src/app/top-banner/badges/launchpad.badge.view.ts [hunk 5]
```diff
@@ -48,18 +53,21 @@ export class LaunchpadBadgeView implements VirtualDOM {
...
+                children: {
+                    policy: 'replace',
+                    source$: state.runningApplications$.pipe(
                         map(
                             (apps) =>
                                 new Set(apps.map((app) => app.cdnPackage)),
                         ),
                     ),
-                    (distinctApps) =>
+                    // checkhere
```
Comment (youwol-guillaume) [inline src/app/top-banner/badges/launchpad.badge.view.ts:64]: // checkhere?

File: src/app/top-banner/customize-dropdown/user-badge-dropdown.view.ts [hunk 6]
```diff
@@ -40,16 +44,20 @@ export class UserBadgeDropdownView implements VirtualDOM {
                 customAttributes: {
                     dataBsToggle: 'dropdown',
                     dataBsAutoClose: 'outside',
+                    ariaExpanded: false,
+                    ariaHaspopup: 'true',
                 },
-                ariaExpanded: false,
-                ariaHaspopup: 'true',
+                // checkthis
```
Comment (youwol-guillaume) [inline src/app/top-banner/customize-dropdown/user-badge-dropdown.view.ts:50]: to check this?

**Review by youwol-guillaume — APPROVED** [review]

**Pull request merged**